.github/workflows/scala-ci.yml

```
@@ -6,6 +6,10 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - spark-version: 3.4.0
+            scala-version: 2.13.10
+          - spark-version: 3.4.0
+            scala-version: 2.12.12
```

```suggestion
            scala-version: 2.12.17
```
https://github.com/apache/spark/blob/v3.4.0/pom.xml#L3593

---

See reason below.